libsolidity/analysis/TypeChecker.cpp

```diff
@@ -356,11 +356,11 @@ void TypeChecker::checkLibraryRequirements(ContractDefinition const& _contract)
 {
 	solAssert(_contract.isLibrary(), "");
 	if (!_contract.baseContracts().empty())
-		m_errorReporter.fatalTypeError(_contract.location(), "Library is not allowed to inherit.");
```

Comment:
Please either leave this fatal or try to crash the compiler in such a situation :-)